src/calculateRentalCost.js

```diff
@@ -4,7 +4,17 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const totalCosts = days * 40;
+
+  if (days >= 7) {
+    return totalCosts - 50;
```

Comment:
Consider defining a constant for the discount value (50) for rentals of 7 or more days to improve code readability and maintainability.